dev-go/go-md2man/go_md2man-1.0.7.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A utility to convert markdown to man pages"
+DESCRIPTION="go-md2man converts standard markdown formatted documents \
+into manpages. It is written purely in Go so as to reduce dependencies \
+on 3rd party libs."
+HOMEPAGE="https://github.com/cpuguy83/go-md2man"
+COPYRIGHT="2017 Brian Goff"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cpuguy83/go-md2man/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="108e4503fed96aa33fc89e98f8cca8f0ac2c3f79d272999463bacdaedf3606ac"
+SOURCE_DIR="go-md2man-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	go_md2man = $portVersion
+	cmd:go_md2man = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
```

Comment:
Please remove this blank line as required by our [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
I can't find where it says "no blank line in BUILD_REQUIRES". Could you point it out?

Reply:
From the [Guidelines][0]

> The correct ordering and spacing of the variables inside the recipe is as follows:

> ```
> BUILD_REQUIRES
> BUILD_PREREQUIRES
> ```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering